README.md

```diff
@@ -4,7 +4,7 @@
 
 <h3 align="center">Build ML pipelines with only Python, run on your laptop, or in the cloud.</h3>
 
-![PyPI](https://img.shields.io/pypi/v/sematic?style=for-the-badge)
+![PyPI](https://img.shields.io/pypi/v/sematic/0.27.0?style=for-the-badge)
```

Comment:
Why this change? Will this be something we need to update every release?
![Screen Shot 2023-03-20 at 7 35 19 AM](https://user-images.githubusercontent.com/7181589/226372832-3d454d10-5821-4eac-8880-d7cc2bd1d6b5.png)

seems to look fine on `main` right now (looks good [here](https://pypi.org/project/sematic/) too)

Reply:
I had written a comment, but in review mode instead of direct comment, so it didn't publish; please see it [here](https://github.com/sematic-ai/sematic/pull/685#discussion_r1140868263).